packages/server/src/db/linkedRows/LinkController.ts

```diff
@@ -172,6 +172,11 @@ class LinkController {
       const rowField = row[fieldName]
       const field = table.schema[fieldName]
       if (field.type === FieldType.LINK && rowField != null) {
+        // Expects an array of docs with at least their _id
+        if (!Array.isArray(rowField)) {
+          throw new Error("Relationship Error: Invalid request")
```

Comment:
Is this an error we expect users to be able to hit, or is this more something you hit during development if you send a request to the API that isn't well-formed?

If it's something the user could reasonably encounter, I think we should try to provide a better error message. Something that includes guidance on how to fix the situation they're in.

Reply:
Users will get this error when pasting invalid content into a relationship cell, like an attachment or a number. This is just a catch-all error for scenarios like that. I used the error on line 230 as a rough template. 

I updated the message to `Relationship Error: Invalid value` to better imply that the pasted data was the issue and not some general issue with the request.